kata-deploy/scripts/install-kata-containerd.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Should the binary files above be excluded? Usually you don't put bins in git. You can download them into the container in the Dockerfile.

Reply:
agreed. removed now.